esp/src/src-react/components/forms/ZAPDialog.tsx

```diff
@@ -103,7 +103,7 @@ const defaultValues: ZAPDialogValues = {
             StartDate: "",
             EndDate: "",
         },
-        RelativeTimeRangeBuffer: "",
+        RelativeTimeRangeBuffer: "43200",
```

Comment:
value corresponds to https://github.com/hpcc-systems/HPCC-Platform/commit/c5da92cb71eb3c8ac935c8fa3fab04bbf9829673